src/ProjectTemplates/BlazorTemplates.Tests/BlazorTemplates.Tests.csproj

```diff
@@ -9,9 +9,6 @@
     <BaseOutputPath />
     <OutputPath />
 
-    <!-- https://github.com/dotnet/aspnetcore/issues/38818 -->
-    <BuildHelixPayload>false</BuildHelixPayload>
-    
```

Comment:
We may want to leave this bit as is for now given this PR is focused on the Playwright upgrade (tests are still broken - missing nav link component issue).